src/components/__internal__/abstract-custom-field/abstract-custom-field.test.jsx

```diff
@@ -71,10 +77,11 @@ describe('AbstractCustomField', () => {
       expect(getByRole('img')).toBeDefined();
     });
 
-    it('gives preference to the error icon', () => {
+    it('renders the error icon first with an error title for accessibility', () => {
       const icon = <Icon name={calendarSvg.id} currentColor="action" title="Hello" />;
       const { getAllByRole } = render(<AbstractCustomField {...sharedProps} icon={icon} errorText="yo" />);
       expect(getAllByRole('img')[0].firstChild).toHaveAttribute('xlink:href', '#icon-caution-fill.svg');
+      expect(getAllByRole('img')[0]).toHaveAttribute('title', 'Error');
```

Comment:
It would be great to refactor this test to use `getByRole('img', { name: 'Error' })` but since it's pre-existing it's just a nice to have. (We only recently found out about `{name: ''}`)